edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/util/ChartModelProvider.java

```diff
@@ -614,7 +614,7 @@ public static Chart createBubbleChart(String chartCaption, String xLabel, List<S
    * @return created chart
    */
   @SuppressWarnings ({"squid:S00107", "squid:S3776"})
-  public static Chart createBubbleChart(String chartCaption, String xLabel, List<String> xTickLabels, String yLabel, List<String> yTickLabels, int xAxisLabelRotation, int yAxisLabelRotation, Double[][] values, Double avgX, Double qX1, Double qX3, Double avgY, Double qY1, Double qY3) {
+  public static Chart createBubbleChart(String chartCaption, String xLabel, List<String> xTickLabels, String yLabel, List<String> yTickLabels, int xAxisLabelRotation, int yAxisLabelRotation, Double[][] values, Double minX, Double maxX, Double minY, Double maxY, Double avgX, Double qX1, Double qX3, Double avgY, Double qY1, Double qY3) {
     // TODO: Tick labels
     // TODO: y serie colors
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)